test/common_utils.py

```diff
@@ -107,6 +107,15 @@ def _check_module_exists(name):
 TEST_WITH_UBSAN = os.getenv('PYTORCH_TEST_WITH_UBSAN', '0') == '1'
 TEST_WITH_ROCM = os.getenv('PYTORCH_TEST_WITH_ROCM', '0') == '1'
 
+# Enables tests that are slow to run (disabled by default)
+TEST_WITH_SLOW = os.getenv('PYTORCH_TEST_WITH_SLOW', '0') == '1'
+
+# Disables non-slow tests (these tests enabled by default)
+# This is usually used in conjunction with TEST_WITH_SLOW to
+# run *only* slow tests.  (I could have done an enum, but
+# it felt a little awkward.
+TEST_SKIP_FAST = os.getenv('PYTORCH_TEST_SKIP_FAST', '0') == '1'
```

Comment:
how about having with_slow and with_fast rather than having them an opposite way? IMO its confusing to have only part of the flags as negations.

Reply:
I negated the flag in recognition of the defaults in question: we default to running fast tests but skipping slow tests. To me, it makes more sense for `FOO=1` to be the non-default case and `unset FOO` or `FOO=` to be the default case.

Reply:
Don't see an issue with default being non zero, but understand it is a personal choice :)